src/rating/rating.spec.ts

```diff
@@ -85,6 +85,33 @@ describe('ngb-rating', () => {
     expect(stars.length).toBe(3);
   });
 
+  it('handles the change event', fakeAsync(() => {
+       const fixture =
+           createTestComponent('<ngb-rating [(rate)]="currentRate" (rateChange)="onRateChange($event)"></ngb-rating>');
```

Comment:
`[(rate)]="currentRate"` is a shortcut for `[rate]="currentRate" (rateChange)="currentRate = $event"`. As such I don't think that we want to have 2 `rateChange` event handlers in this test.

More generally, IMO the simplest possible test for the issue we are trying to solve here could be:

```typescript
  it('should now fire change event initially',  fakeAsync(() => {
    const fixture = createTestComponent('<ngb-rating [rate]="3" (rateChange)="changed = true"></ngb-rating>');
    tick();
    expect(fixture.componentInstance.changed).toBeFalsy();
  }));
```